---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Cell In[4], line 3
      1 import numpy as np
      2 from PIL import Image
----> 3 import cv2
      5 def load_image(image_path):
      6     """
      7     Load an image from a file.
      8     :param image_path: Path to the image file.
      9     :return: PIL Image object.
     10     """

File ~/.python/current/lib/python3.12/site-packages/cv2/__init__.py:181
    176             if DEBUG: print("Extra Python code for", submodule, "is loaded")
    178     if DEBUG: print('OpenCV loader: DONE')
--> 181 bootstrap()

File ~/.python/current/lib/python3.12/site-packages/cv2/__init__.py:153, in bootstrap()
    149 if DEBUG: print("Relink everything from native cv2 module to cv2 package")
    151 py_module = sys.modules.pop("cv2")
--> 153 native_module = importlib.import_module("cv2")
    155 sys.modules["cv2"] = py_module
    156 setattr(py_module, "_native", native_module)

File ~/.python/current/lib/python3.12/importlib/__init__.py:90, in import_module(name, package)
     88             break
     89         level += 1
---> 90 return _bootstrap._gcd_import(name[level:], package, level)

ImportError: libGL.so.1: cannot open shared object file: No such file or directory